test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -126,6 +126,31 @@ public async Task WorkerIndexing_Setting_ChannelInitializationState_Succeeds()
             Assert.Equal(expectedProcessCount, initializedChannelsCount);
         }
 
+        [Fact]
+        public async Task WorkerRuntime_Setting_ChannelInitializationState_Succeeds()
+        {
+            _testLoggerProvider.ClearAllLogMessages();
+            int expectedProcessCount = 3;
+            RpcFunctionInvocationDispatcher functionDispatcher = GetTestFunctionDispatcher(expectedProcessCount, false, runtime: RpcWorkerConstants.JavaLanguageWorkerName, workerIndexing: true);
+
+            var testEnv = new TestEnvironment();
+            testEnv.SetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime, null);
```

Comment:
I do not see `testEnv` used anywhere ? instead simplify - do not pass `runtime: RpcWorkerConstants.JavaLanguageWorkerName`  on line 134 - this will ensure environment variable is not set

Reply:
I tried that way but getting error. RpcFunctionInvocationDispatcher constructor needs workerruntime as environment variable. In line 134, method GetTestFunctionDispatcher() internally calls the constructor of RpcFunctionInvocationDispatcher.

Reply:
Corrected it. Thanks